### PR TITLE
scaffold searchform

### PR DESCRIPTION
**Pull request by mikeyrayvon**
move nav wrapper out of nav partial. add search form. get search formin header

### CODE REVIEW

File: searchform.php [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<form role="search" method="get" id="searchform" class="searchform" action="<?php echo esc_url( home_url( '/' ) ); ?>">
+  <div>
+    <input type="text" class="text-align-right" value="<?php echo get_search_query(); ?>" name="s" id="s" />
+    <input type="submit" id="searchsubmit" value="0" />
```
Comment (varffs) [inline searchform.php:4]: why a 0 for the button?

**Comment by varffs** [comment]
why did u call it scaffold searchform when most of the pr is something else?

File: searchform.php (hunk 1 above)
Comment (mikeyrayvon) [inline searchform.php:4]: its a placeholder.  its supposed to be a magnifying glass but i dont have the icon from clay yet

**Comment by mikeyrayvon** [comment]
everything else has to do with the scaffolding of the searchform.  i had to make room for it in the row.  why u trippin?

**Pull request merged**